lib/game.rb

```diff
@@ -1,29 +1,23 @@
 module TicTacToe
 
   class Game 
-    TTT_MARKER1 = "X"
-    TTT_MARKER2 = "O"
-    
-    def initialize(game_turn, marker_loop, game_completion, game_eval)
+    attr_reader :player1
+    attr_reader :player2
+
+    def initialize(game_turn, game_eval, p1_marker, p2_marker)
       @game_turn = game_turn
-      @marker_loop = marker_loop
-      @game_completion = game_completion
       @game_eval = game_eval
+      @player1 = Player.new(1, p1_marker)
+      @player2 = Player.new(2, p2_marker)
     end
 
-    def run 
-      marker1, marker2 = @marker_loop.marker_selection(TTT_MARKER1, TTT_MARKER2)
-
-      @player1, @player2 = Player.new(1, marker1), Player.new(2, marker2)
-
+    def players_turns
       current_player = @player1
-
       begin 
         board_array = @game_turn.execute(current_player.ord_num, current_player.marker)
         current_player = toggle_player(current_player)
-      end until @game_eval.game_over?(board_array, TTT_MARKER1, TTT_MARKER2)
-      
-      @game_completion.game_over_messages(board_array, @player1, @player2)
+      end until @game_eval.game_over?(board_array, @player1, @player2)
```

Comment:
@avni510 Great point. My previous comment was misleading in saying that GameEval would be isolated from future changes to Player. I was only thinking in terms of that particular method's implementation. Your comment "by changing line 19 I still won't completely get rid of the dependency that GameEval has on Player (although I'll be able to reduce it)" is indeed correct 👍 

When I looked at `#game_over?` there didn't seem to be any benefits of passing in the player objects. In the case of `#player_won`, you actually return the player object, so at a quick glance it seems like the object is at least necessary for the current method's return type to stay consistent.